catalog/core/catalog-core-metacardgroomerplugin/src/main/java/ddf/catalog/plugin/groomer/metacard/StandardMetacardGroomerPlugin.java

```diff
@@ -72,6 +73,15 @@ protected void applyCreatedOperationRules(CreateRequest createRequest, Metacard
             aMetacard.setAttribute(new AttributeImpl(Metacard.TAGS,
                     Collections.singletonList(Metacard.DEFAULT_TAG)));
         }
+
+        // upon creation of the metacard, generate attributes for created, modified dates
```

Comment:
My thought was that since resource descriptors can have a life before they enter the system, there would be value in preserving an initial modified date if one is set. I don't know how queries work with versioning though, so it's possible that created is the only one that matters.